includes/discovery/sensors/count/christie-projector.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
```

Comment:
Any reason this is not in yaml?

Reply:
Hi.

To be honest, I copied the epson-projector model and adapted to my needs.

I'll check this week or next week how to to it in YAML.

Thanks.